shell/components/Questions/index.vue

```diff
@@ -258,7 +258,7 @@ export default {
       }
 
       if ( this.tabbed === 'multiple' ) {
-        return this.groups.length > 1;
+        return this.groups.length;
```

Comment:
If I was being picked I'd add `!!` to return a boolean rather than rely on a truthy/falsey

Reply:
you are correct @nwmac . Fixed